### PR TITLE
Add filetype filters and css fix

### PR DESCRIPTION
**Pull request by williamputraintan**
- add to support html and png files
- add filter for WTS and WGS
- aligned-center image when open in preview
- Add a maximum fileSize for the `/files` endpoint

### CODE REVIEW

File: src/containers/FileViewer.tsx [hunk 1]
```diff
@@ -203,7 +227,7 @@ function FileViewer() {
           </Grid>
           <Grid item>
             <Typography variant='h6'>
-              {subjectId} - Image Analysis Output: {pipelineSelection.name}
+              {subjectId} - File Analysis Output (PNG and HTML): {pipelineSelection.name}
```
Comment (victorskl) [inline src/containers/FileViewer.tsx:230]: Can you pls change to format like -
```
{subjectId} Analysis Output: {pipelineSelection.name}
```

File: src/containers/Subject.js [hunk 2]
```diff
@@ -686,7 +686,7 @@ class Subject extends Component {
             <ListItemIcon>
               <PhotoLibraryIcon color={'action'} />
             </ListItemIcon>
-            <ListItemText primary='View Analysis Output Images' />
+            <ListItemText primary='View Analysis Output (PNG and HTML)' />
```
Comment (victorskl) [inline src/containers/Subject.js:689]: Also we can format it to -

```
<ListItemText primary='View Analysis Output in File Viewer' />
```

**Review by victorskl — APPROVED** [review]
A tiny ... bit sceptical about viewing embedded html here. But, hey, I could be wrong. Let's experiment it! :)

**Comment by williamputraintan** [comment]
> A tiny ... bit sceptical about viewing embedded html here. But, hey, I could be wrong. Let's experiment it! :)

Yup, we could remove easily if needed!

**Comment by victorskl** [comment]
Yup, yup!

**Pull request merged**